Device/Device/doc/spec.md

```diff
@@ -153,6 +153,10 @@ Obviously, in order to toggle the referred switch, this attribute value will hav
     + Attribute type: [DateTime](https://schema.org/DateTime)
     + Optional    
 
++ `owner` : Entity's owners: an array of entities that represents the owners of an Thing.
```

Comment:
The owner of a thing or the owner of the device?

Reply:
in general of an object (for the common schema), for the specific data model of a device